distributions/openhab/src/main/resources/bin/update.ps1

```diff
@@ -112,6 +112,10 @@ Function Update-openHAB {
         Remove-Item ($userdata + '\cache') -Recurse -ErrorAction SilentlyContinue
         Remove-Item ($userdata + '\tmp') -Recurse -ErrorAction SilentlyContinue
 
+        # We need to keep a backup in case the user modified that file
+        Copy-Item ($userdata + '\etc\org.ops4j.pax.url.mvn.cfg') -Destination ($userdata + '\etc\org.ops4j.pax.url.mvn.cfg.bak') -Force
+        Remove-Item ($userdata + '\etc\org.ops4j.pax.url.mvn.cfg') -ErrorAction SilentlyContinue
```

Comment:
Mentioned in a separate comment, but thought i'd post as a review too: This is a different file to the logging file used in bin/update